app/models/user.rb

```diff
@@ -37,6 +37,11 @@ def self.new_with_session(params, session)
     end
   end
 
+  def self.search(input, search)
+    input.where("firstname LIKE ?", "%#{search}%")
+  end
```

Comment:
Yeah I was trying to search both the order model as well as the user model but you're right for now this doesnt work